src/maintainer/infrastructure.rst

```diff
@@ -19,11 +19,11 @@ Smithy contains maintenance code for conda-forge, which is used by the ``conda-s
 
 ``conda-forge/conda-smithy`` is the right repository to report bugs for
 
- - the rerendering process
- - the recipe linter
+ - The rerendering process
+ - The recipe linter
  - :term:`CI` support utils
 
-``conda-smithy`` also contains the command line tool that you will use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
+``conda-smithy`` also contains the command line tool that you can use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
```

Comment:
It's the only tool we use to re-render (particularly when doing this locally). So maybe "should" is better

```suggestion
``conda-smithy`` also contains the command line tool that you should use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
```